requirements.txt

```diff
@@ -1,11 +1,13 @@
 pandas>=1.3.0
 numpy>=1.16.5
 requests>=2.31
+requests_cache==1.1.0
+requests_ratelimiter==0.4.2
 multitasking>=0.0.7
 lxml>=4.9.1
 appdirs>=1.4.4
 pytz>=2022.5
 frozendict>=2.3.4
 beautifulsoup4>=4.11.1
 html5lib>=1.1
-peewee>=3.16.2
\ No newline at end of file
+peewee>=3.16.2
```

Comment:
revert changes

Reply:
I reverted the explicit dependencies but kept the newline at the end.